compiler/aarch64/codegen/FPTreeEvaluator.cpp

```diff
@@ -257,54 +257,71 @@ OMR::ARM64::TreeEvaluator::dnegEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
 	}
 
+static TR::Register *
+intFpTypeCoversionHelper(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
```

Comment:
Could you fix the spelling of "Conversion" please, and then fix up the calls throughout?

Reply:
Corrected spelling.